packages/liveblocks-react/src/config.ts

```diff
@@ -11,4 +11,7 @@ export const config = {
 
   USER_THREADS_POLL_INTERVAL: 1 * MINUTES,
   USER_THREADS_MAX_STALE_TIME: 5 * SECONDS,
+
+  HISTORY_VERSIONS_POLL_INTERVAL: 1 * MINUTES,
+  HISTORY_VERSIONS_MAX_STALE_TIME: 3 * SECONDS,
```

Comment:
Any reason to not use 5 seconds here?

Reply:
This is supposed to be 5 seconds 😅 . It slipped from my example testing. Thanks for pointing out.